card/lib/card.rb

```diff
@@ -4,20 +4,10 @@
 
 class Card < ActiveRecord::Base
   require_dependency 'card/active_record_ext'
-  require_dependency 'card/codename'
   require_dependency 'card/query'
-  require_dependency 'card/set_pattern'
-  require_dependency 'card/set'
-  require_dependency 'card/format'
   require_dependency 'card/exceptions'
-  require_dependency 'card/auth'
   require_dependency 'card/log'
-  require_dependency 'card/loader'
-  require_dependency 'card/content'
   require_dependency 'card/action'
-  require_dependency 'card/act'
-  require_dependency 'card/change'
-  require_dependency 'card/reference'
```

Comment:
I'm a bit wishy-washy on this too.  The fact is that is one of the uglinesses of rails autoloading, and that is why there needed to be a require for Card::Set in the loader.  Otherwise it ends up loading the global Set into Card.

I guess I lean a little towards leaving them out until a problem develops, but I could easily be swayed the other way.  Probably, we shouldn't include things like the history classes (card/act|action|change) because Card shouldn't depend on them directly because history may load a whole other set of classes.